commcare_export/builtin_location_types_query.py

```diff
@@ -0,0 +1,155 @@
+                    {
+                        "Lit": "resource_uri"
+                    },
+                    {
+                        "Lit": "shares_cases"
+                    },
+                    {
+                        "Lit": "view_descendants"
+                    }
+                ],
+                "source": {
+                    "Map": {
+                        "body": {
+                            "List": [
+                                {
+                                    "Ref": "id"
+                                },
+                                {
+                                    "Ref": "administrative"
+                                },
+                                {
+                                    "Ref": "code"
+                                },
+                                {
+                                    "Ref": "domain"
+                                },
+                                {
+                                    "Ref": "name"
+                                },
+                                {
+                                    "Ref": "parent"
+                                },
+                                {
+                                    "Ref": "resource_uri"
+                                },
+                                {
+                                    "Ref": "shares_cases"
+                                },
+                                {
+                                    "Ref": "view_descendants"
+                                }
+                            ]
+                        },
+                        "source": {
+                            "Apply": {
+                                "fn": {
+                                    "Ref": "api_data"
+                                },
+                                "args": [
+                                    {
+                                        "Lit": "location_type"
+                                    },
+                                    {
+                                        "Ref": "checkpoint_manager"
+                                    }
+                                ]
+                            }
+                        },
+                        "name": None
+                    }
+                },
+                "missing_value": None
+            }
+        }
+    }
+}
+
+
+def build_dictionary(location_types):
+    # Convert to a dictionary of dictionaries, keyed by location type
+    # resource URI
+    location_type_dict = {}
+    headings = location_types[TABLE_NAME]['headings']
+    rows = location_types[TABLE_NAME]['rows']
+    for row in rows:
+        resource_uri = None
+        row_dict = {}
+        assert len(row) == len(headings)
+        for i in range(0, len(headings)):
+            row_dict[headings[i]] = row[i]
+            if headings[i] == 'resource_uri':
+                resource_uri = row[i]
```

Comment:
```suggestion
        row_dict = dict(zip(headings, row))
        resource_uri = row_dict.get('resource_uri')
```

Reply:
Done, and the code moved to location_info_provider.py.